src/Sylius/Bundle/ApiBundle/Resources/config/validation/Address.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+
+    <class name="Sylius\Component\Core\Model\Address">
+        <property name="countryCode">
+            <constraint name="NotNull" />
+        </property>
+        <property name="street">
+            <constraint name="NotNull" />
+        </property>
+        <property name="city">
+            <constraint name="NotNull" />
+        </property>
+        <property name="postcode">
+            <constraint name="NotNull" />
+        </property>
+        <property name="provinceName">
+            <constraint name="NotNull" />
+        </property>
+    </class>
+</constraint-mapping>
```

Comment:
```suggestion
```
This file is not needed at all. We should reuse the existing validation file. You should add `<attribute name="validation_groups">sylius</attribute>` to Address resource to use default validation